substrafl/remote/register/register.py

```diff
@@ -85,7 +85,9 @@ def _copy_local_packages(
     for dependency_path in local_dependencies:
         dest_path = path / dependency_path.name
         if dependency_path.is_dir():
-            shutil.copytree(dependency_path, dest_path)
+            shutil.copytree(
+                dependency_path, dest_path, ignore=shutil.ignore_patterns("local-worker*", "tmp_substrafl*")
```

Comment:
Ignoring local-worker is a substrafl solution for a substra problem, maybe it should be solved there

Reply:
Done in https://github.com/Substra/substra/pull/314 :)